templates/add_new_item.html

```diff
@@ -0,0 +1,35 @@
+{% extends "layout.html" %}
+{% block title %}Add To-Do Item{% endblock %}
+{% block content %}
+
+<div class="jumbotron">
+   <p class="lead">Form to add new To-Do Item</p>
+    <form action="/add" method="post">
+      <Table>
+          <tr style="align:middle">
+              <td> <label for="title">To-Do Item Title</label> </td>
+              <td> <input name="title" id = "title"> </td>
+          </tr>
+          <tr>
+              <td> <label for="description">To-Do Item Description</label> </td>
+              <td> <textarea name="description" id="description">Enter text here...</textarea> </td>
```

Comment:
You'll want to use a `placeholder` attribute rather than entering in default text here